packages/static-wado-webserver/lib/dicomWebServerConfig.mjs

```diff
@@ -10,11 +12,23 @@ const { staticWadoConfig } = StaticCreator;
 const { dicomWebServerConfig } = ConfigPoint.register({
   dicomWebServerConfig: {
     configBase: staticWadoConfig,
+    dicomWebScpConfig,
     helpShort: "dicomwebserver",
     helpDescription: "Serve up the static wado files and optionally a web client as a web server on the local machine.",
     studyQuery: "studiesQueryByIndex",
     clientDir: "~/ohif",
     port: 5000,
+    externalViewer: {
+      protocol: "http",
+      host: "localhost",
+      port: 3000,
+    },
+    sourceScpConfig: {
```

Comment:
This should just reference the ae title name, something like:
proxyAe: "CLEARCANVAS"
Also, it should probably be defined in staticWadoConfig, because the proxyAe should apply to both SCP and Web Server components.

Reply:
To be honest I'm still confused about this configuration thing. More it grow less control we have.
But I will move to the place you said as it makes sense to you (to me also, at least for now)

Reply:
The configuration allows for very deep changes, and that is required in order to have custom settings at a site where entire parts of the functionality get replaced, and I've used that type of configuration setup elsewhere where different types of configurations got run, and had it work out well even if the configuration can be complex.